component-studio/component-runtime-di/src/test/java/org/talend/sdk/component/runtime/di/record/DiRowStructVisitorTest.java

```diff
@@ -15,21 +15,6 @@
  */
 package org.talend.sdk.component.runtime.di.record;
 
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.talend.sdk.component.api.record.SchemaProperty.IS_KEY;
-import static org.talend.sdk.component.api.record.SchemaProperty.SIZE;
-import static org.talend.sdk.component.api.record.SchemaProperty.PATTERN;
-import static org.talend.sdk.component.api.record.SchemaProperty.SCALE;
-import static org.talend.sdk.component.api.record.SchemaProperty.STUDIO_TYPE;
-
```

Comment:
revert import changes.

Reply:
statics should be in first according component-runtime style rules based on those mentioned above.